driver-adapters/neon-http-lambda-basic/run.sh

```diff
@@ -3,7 +3,6 @@
 set -eu
 
 pnpm install
-pnpm prisma generate
```

Comment:
This was needed, I believe, because the dependencies are sometimes cached in the github job and that causes the client no to generate and then causing flakiness.

Reply:
Looks like a typo, I didn't want to remove that one